test/unit/utils/install_context_test.rb

```diff
@@ -2,17 +2,17 @@
 require "inspec/globals"
 require "inspec/utils/install_context"
 
-def check_install_contexts(test_expected_to_be_true, also_rubygem)
+def assert_install_contexts(test_expected_to_be_true, also_rubygem)
   should_be_false = %w{chef-workstation chefdk docker
                      habitat omnibus rubygem source}
   should_be_false -= [test_expected_to_be_true]
   should_be_false = should_be_false.map { |m| "#{m}_install?".tr("-", "_").to_sym }
   should_be_false -= [:rubygem_install?] if also_rubygem
-  should_be_false.each { |m| expect(Inspec.method(m).call).must_equal false }
+  should_be_false.each { |m| expect(Inspec.send(m)).must_equal false }
 
   should_be_true   = ["#{test_expected_to_be_true}_install?".tr("-", "_").to_sym]
   should_be_true  += [:rubygem_install?] if also_rubygem
-  should_be_true.each { |m| expect(Inspec.method(m).call).must_equal true }
+  should_be_true.each { |m| expect(Inspec.send(m)).must_equal true }
```

Comment:
How do you debug when all it says is`Expected true, got false` ?

Compare to `Expected Inspec to be docker` (these names aren't exactly adjectives... so it's a little clunky)

Reply:
As I described above, your suggestion is not an equivalent test. We are testing predicates with expected boolean results, not referencing a method on a module.

When a test fails, it's easy to see which one - I've forced a failure here:
```
Inspec::InstallContextHelpers::when it looks like a Habitat installation#test_0001_should properly detect a habitat install [/Users/cwolfe/sandbox/inspec/inspec-telemetry-2/test/unit/utils/install_context_test.rb:53]:
Expected: false
  Actual: true
```

I do wish I had a better expectation - perhaps one that could take a custom message, in which case I could provide user context. But this is not worthwhile at this point; it's a simple matter to add a pry statement should it ever break.